data/modules/Scout/Scout.lua

```diff
@@ -303,24 +303,30 @@ local function calcOrbitalScanMission(sBody, difficulty, reward)
 
 	local resolutionScalar = math.invlerp(p.resolution_min, p.resolution_max, resolution)
 
+	local body_coverage = coverage
 	if radiusScalar > 1.0 then
 		-- body is small, increase coverage of the scan
-		coverage = math.min(coverage, 1.0) * (radiusScalar ^ 0.9)
+		body_coverage = math.min(coverage, 1.0) * (radiusScalar ^ 0.9)
 	else
 		-- body is large, reduce coverage of the scan proportionally
-		coverage = math.min(coverage, 1.0) * radiusScalar
+		body_coverage = math.min(coverage, 1.0) * radiusScalar
 		-- similarly increase the resolution of the scan to ensure we can scan at higher altitudes
 		resolution = resolution * (1.0 + math.log(radiusKm / p.nominal_radius, 10))
 	end
 
-	coverage = math.min(coverage * realDifficulty, 1.0)
+	-- adjust coverage against difficulty and clamp to a maximum of 100%
+	body_coverage = math.min(body_coverage * realDifficulty, 1.0)
+
+	-- finally, convert the coverage into km2 and limit the number of decimals
+	local body_area = 4 * math.pi * radiusKm^2
+	local coverageKm2 = utils.round(body_area, 0.0001)
 
 	local rewardAmount = reward
 		* p.reward_per_km * (math.pi * radiusKm * coverage)
```

Comment:
Reward needs to be updated to use `math.sqrt(coverageKm2)` rather than `pi * radiusKm * coverage`.

Additionally, `reward_per_km` may need to be adjusted in the `orbital_params` definition to scale mission rewards back into line. Quick playtesting with the requested changes suggests it should be increased by up to 20% to make small moon scans viable and reward difficult low-orbit scans around large bodies.

Reply:
Fixed in c69f89755327d169f7256c9a47dba352263120d1

Anecdotally, the rewards seem to be significantly less now..

Reply:
I'm noticing that as well. For now, let's set `reward_per_km = 0.11`, and `reward_resolution_min = 12.0` - in testing, this seems to produce adequately-lucrative results. A follow-on PR should adjust `MissionUtils.CalcReward` to take into account 1) this is a two-way trip rather than a single-hop to a station, and 2) properly calculate the distance between the closest hyperspace jump-in point and the target body in a distant system.

Reply:
The rewards for these missions (before this PR) seem to be fairly high, given there's no risk and quite often are even in the same system. As soon as a player can afford the scanners this seems to be the most lucrative mission type for starting-out players (based on my rather limited playtime so far..)

Reply:
Updated rewards in d6d1f690a0d76a1c17bae0b6496ce81c14e8e6c3

Reply:
> A future PR

These are all great ideas - I would hate for them to get lost. Please feel free to add them as a comment header to `Scout.lua`, or document them on the forum at your discretion.

Reply:
> > A future PR
> 
> These are all great ideas - I would hate for them to get lost. Please feel free to add them as a comment header to `Scout.lua`, or document them on the forum at your discretion.

Added in b8d008bd3efb92b527a5ed552d95926996383300